package.json

```diff
@@ -47,9 +50,11 @@
     "mocha": "^6.1.4",
     "mongoose": "^5.5.7",
     "morgan": "^1.9.1",
+    "protobufjs": "^6.8.8",
```

Comment:
Remove this package since it's not used.

Reply:
Done.